### DISCUSSION TITLE
can i hide some url in docs? somes like  /graphql

### DISCUSSION THREAD

**Discussion by goodking-bq** [fastapi/fastapi#7295] [Questions] [answered]
quest easy ,see title . thanks.

**Comment by SirTelemak** [comment, marked as answer]
Yeah, there is param for router `include_in_schema`
Example:
```python3
app = FastAPI()
@app.get('/', include_in_schema=False)
def foo():
    return {'success': True}
```

**Comment by iudeen** [comment]
> Yeah, there is param for router `include_in_schema`
> Example:
> ```python3
> app = FastAPI()
> @app.get('/', include_in_schema=False)
> def foo():
>     return {'success': True}
> ```

This is the perfect answer! How about we close this issue?

**Comment by goodking-bq** [comment]
> > Yeah, there is param for router `include_in_schema`
> > Example:
> > ```python
> > app = FastAPI()
> > @app.get('/', include_in_schema=False)
> > def foo():
> >     return {'success': True}
> > ```
> 
> This is the perfect answer! How about we close this issue?

yes .thanks.i see it just now.

**Comment by tiangolo** [comment]
Thanks for the help here @SirTelemak ! :clap: :bow: 

Thanks for reporting back and closing the issue :+1:
